### ISSUE TITLE
error gui

### ISSUE THREAD

**Issue by xxsle** [EliaFantini/APS-Aerial-Photography-Simulation-for-photogrammetric-flight-design#2]
I opened the GUI but I am getting this error. what should I do? I would be very happy if you help me :( that's why I can't get any results.

![jnjöçş_ss](https://github.com/EliaFantini/APS-Aerial-Photography-Simulation-for-photogrammetric-flight-design/assets/108673936/ef7df5a2-32d7-4379-b3a8-552cfbd8ecb4)

**Comment by EliaFantini** [comment]
Hi! Sorry for the late reply, I was very busy last week. Did you solve already? From the screen you sent it looks like the input provided to rasterio library is not valid, so maybe the files you loaded in the program are not supported by rasterio, or encoded differently than expected? It seems that it tries to iterate on xs but xs is not iterable

**Comment by xxsle** [comment]
hello. actually I tried to install the rasterio library as you described, but I don't understand where I made a mistake. the input files I uploaded to the program are actually the ones you shared

**Comment by EliaFantini** [comment]
As I wrote in the readme: "The dataset in the Data folder contains a compressed version of the original orthophoto file, which originally was 200MB. There was no GeoTIFF compressor available online so it is put there just to be visualized but it won't work in the software as it lacks the geographical informations which make a TIFF image a GeoTIFF image. If anyone would like to try the software on my dataset feel free to open an issue or contact me and I will provide a cloud link to download the whole file."
So if you used those files it is expected that it doesn't work. I'll look for the complete dataset in the next days when I'll have access to the pc I worked this project on, hoping they are still there
